sass/_post.scss

```diff
@@ -0,0 +1,81 @@
+.post__header__logo {
+  height: 80px;
+  margin: 0 20px;
+  border-bottom: 2px solid #fff;
+  @media screen and (max-width: 680px) {
+      height: 40px;
+      border-bottom: none;
+  }
+}
+.post__header {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  width: 50%;
+  background-image: url('./images/hunkahunkakittylove.jpg');
+  background-size: cover;
+  background-position: 50% 50%;
+  background-color: #DCBF79;
+  color: $text_light;
+  > h1 {
+    font: $header_font;
+    font-size: 2em;
+    padding-left: 20px;
+    margin-bottom: 40px;
+  }
+  @media screen and (max-width: 680px) {
+      position: absolute;
+      bottom: 60%;
+      margin-bottom: 5px;
+      width: 100%;
+  }
+}
+.post__headerNav {
+  list-style: none;
+  margin-top: 15px;
+  padding-left: 20px;
+  > li {
+    display: inline-block;
+      &:after {
```

Comment:
these don't need to be indented within `display`. Backing these up 2 spaces is correct